tests/constraints/models.py

```diff
@@ -72,16 +72,20 @@ class Meta:
 class UniqueConstraintProduct(models.Model):
     name = models.CharField(max_length=255)
     color = models.CharField(max_length=32, null=True)
+    age = models.IntegerField(null=True)
 
     class Meta:
         constraints = [
             models.UniqueConstraint(
                 fields=["name", "color"],
                 name="name_color_uniq",
-                # Custom message and error code are ignored.
+            ),
+            models.UniqueConstraint(
+                fields=["color", "age"],
+                name="color_age_uniq",
                 violation_error_code="custom_code",
                 violation_error_message="Custom message",
-            )
+            ),
```

Comment:
If that can make testing any easier the constraints don't need to be defined on the model itself anymore to be testable since 13922580cccfb9ab2922ff4943dd39da56dfbd8c.